server/db/models/product.js

```diff
@@ -1,6 +1,6 @@
 const Sequelize = require('sequelize')
 const db = require('../db')
-const {STRING, BOOLEAN, INTEGER, FLOAT} = Sequelize
+const {STRING, BOOLEAN, INTEGER, FLOAT, JSON} = Sequelize
```

Comment:
Let's remove the JSON destructuring since we aren't using it in the file.

Reply:
yes, accidentally left that there...